pkg/qemu/qemu.go

```diff
@@ -341,6 +345,15 @@ func Cmdline(cfg Config) (string, []string, error) {
 		return "", nil, err
 	}
 
+	version, err := getQemuVersion(exe)
+	if err != nil {
+		return "", nil, err
+	}
+	logrus.Debugf("QEMU version %s detected", version.String())
+	if version.LessThan(*semver.New(MinimumQemuVersion)) {
+		logrus.Fatalf("QEMU %v is too old", version)
```

Comment:
The minimum version has to be included in the error (or the warning) too

Reply:
We should return an error if the version isn't 0.0.0. In that case the parsing didn't fail, but the version is too low; so we know it will break.

(sorry, commented in the wrong place originally)

Reply:
The fatal logging will also terminate the program

Reply:
> The fatal logging will also terminate the program

Oh, right, forgot about that.

I guess this means it will abort even when parsing failed because then the version will be `0.0.0`. I think this is fine, but it sounded like @AkihiroSuda didn't want to abort in that case, so you would have to downgrade the Fatal to Error.

Reply:
Added some error handling and unit tests, to the version parsing